mitodl/bootcamp-ecommerce
# Implemented order receipt/cancellation message

#### What are the relevant tickets?
Fixes #39 

#### What's this PR do?
Pops up a modal when there's a successful payment, when the user cancelled the payment, and when two minutes have passed without determining if the payment went through.

#### How should this be manually tested?
- Go to `/pay/?status=cancel` and you should see the cancellation message
- Make a payment, then get a reference to the `Order` in the shell. Change the status from `fulfilled` to `created` and save.
- Go to `/pay/?status=receipt&order=order_num` replacing `order_num` with `order.id` and look at the console logs. You should see it refetching the klass API every 3 seconds or so. 
- In the shell change the status back to `fulfilled` and resave the order. After 3 seconds or so you should see a success toast message pop up.
- If you waited two minutes without changing the order status you should see a failure message and no more attempts at refetching the klasses API.
